### DISCUSSION TITLE
Not able to save list of commands in batch mode

### DISCUSSION THREAD

**Discussion by aborruso** [saulpw/visidata#1209] [General]
Hi,
I open vd, then I open the list of commands (`z CTRL+H` ), I save it as CSV and then I save this batch:

```
sheet	col	row	longname	input	keystrokes	comment
global		null_value	set-option			
._commands			save-sheet	visiDataCommands.csv	Ctrl+S	save current sheet to filename in format determined by extension (default .tsv)
```

If I run `vd -b -p visiDataCommands.vd`, I have 

```
opening visiDataCommands.vd as vd
no sheet named ._commands
replay canceled
```

I think that the problem is that I do not have `z CTRL+H`, in my batch file. Why is it not there? Am I doing something wrong?

I'm using `saul.pw/VisiData v2.8dev`

Thank you

**Comment by saulpw** [comment]
It's because the `help-commands` command is not logged; see the `logged` column if you use `gv` on that commands sheet and look for that command.  I think we need to do something different in this case, so the sheet creation command is logged regardless of whether or not it's logged normally.

**Reply by aborruso** [reply]
I'm building a pipeline to translate it in Italian, to create by script a documentation page for my guide. 

My initial steps ...

```
- longname: error-cell
  sheet: TableSheet
  keystrokes: zCtrl+E
  description: view traceback for error in current cell
  lang: it
  description_it: 'visualizza il traceback dell''errore nella cella corrente'
```
